tests/integration/test_deploy_model_ssl_off_auth_off.py

```diff
@@ -19,7 +19,7 @@ def test_deploy_ssl_off_auth_off(self):
         self.assertEqual(200, SentimentAnalysis_request.status)
         SentimentAnalysis_request.read()
 
-        conn.request("GET", "/endpoints/tTest")
-        tTest_request = conn.getresponse()
-        self.assertEqual(200, tTest_request.status)
-        tTest_request.read()
+        conn.request("GET", "/endpoints/ttest")
```

Comment:
this code is repeated for 3 times now. Can we keep configurations separately (in a dict?) and just run the same code for every configuration?

Reply:
fixed